docker-compose.yml

```diff
@@ -2,17 +2,20 @@ version: '3'
 services:
   web:
     build: web/
-    image: helpbuttons/helpbuttons-web:main
+ #   image: helpbuttons/helpbuttons-web:main
+    volumes: 
```

Comment:
please revert everything in this file. each PR has to be a single feature. im fixing this in another branch ;)